{{cookiecutter.project_slug}}/requirements/local.txt

```diff
@@ -8,7 +8,7 @@ psycopg2==2.9.3  # https://github.com/psycopg/psycopg2
 psycopg2-binary==2.9.3  # https://github.com/psycopg/psycopg2
 {%- endif %}
 {%- if cookiecutter.use_async == 'y' or cookiecutter.use_celery == 'y' %}
-watchgod==0.8.2  # https://github.com/samuelcolvin/watchgod
```

Comment:
I was going to ask if we still need watchgod for uvicorn, but it looks like support for watchfiles [was added](https://github.com/encode/uvicorn/pull/1437) in [v0.18.0](https://github.com/encode/uvicorn/releases/tag/0.18.0).